proto/cosmos/upgrade/v1beta1/upgrade.proto

```diff
@@ -40,6 +40,35 @@ message Plan {
   // If this field is not empty, an error will be thrown.
   google.protobuf.Any upgraded_client_state = 5
       [deprecated = true, (gogoproto.moretags) = "yaml:\"upgraded_client_state\""];
+
+  // Optional: Upgrade contains additional instructions for the devops or a hypervisor.
+  // App specific instructions are handled by the `info` attribute. Here we provide
+  // information such as pre-upgrade or post-upgrade commands.
+  UpgradeInstructions Upgrade = 6;
```

Comment:
If you think about a better name for this attribute, please share your proposals. I was also thinking about `Setup`, `Preparation` ...
We should not confuse with the `Info` field which is already overloaded (both in meaning and usage).

Reply:
Considering it contains both pre and post upgrade info, `UpgradeInstructions` sounds good.